### DISCUSSION TITLE
Welcome to k-wave-python Discussions!

### DISCUSSION THREAD

**Discussion by waltsims** [waltsims/k-wave-python#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ryancinsight** [comment]
@waltsims I saw that you starred kwavers, It's not impressive yet but I'm trying to set the foundation for physics informed neural networks for therapy and imaging. Trying to support generic backends through wgpu as well. Happy to collaborate if you are interested. Best regards

**Reply by waltsims** [reply]
I'm a fan of the direction! Excited to see the project grow.
